core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -327,10 +323,11 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
     private void checkGraalVMVersion(String version) {
         log.info("Running Quarkus native-image plugin on " + version);
         final List<String> obsoleteGraalVmVersions = Arrays.asList("1.0.0", "19.0.", "19.1.", "19.2.");
-        final boolean vmVersionIsObsolete = obsoleteGraalVmVersions.stream().anyMatch(v -> version.contains(" " + v));
+        final boolean vmVersionIsObsolete = version.contains(" 19.3.0 ")
```

Comment:
Is the extra space in the beginning of the string really needed?

Reply:
It's to be on the safe side so that we are sure it's the start of the version.